0020-bluetooth-payment-protocol/0020-bluetooth-payment-protocol.md

```diff
@@ -0,0 +1,110 @@
+# Bluetooth Payment Protocol
+
+This document specifies a protocol for communicating and authorizing Interledger Payment Requests over Bluetooth (Low Energy). It is primarily intended for use in in-person retail payment contexts.
+
+This protocol may be implemented by Customer devices such as smartphones or very low-power, offline devices such as [tapFi](https://github.com/joaopedrovbs/tapFi).
+
+Once a Customer becomes active by broadcasting it's payment service over BLE, Merchants can connect
+and request signature for payments over a secure Bluetooth channel. The channel is closed once
+a payment is signed or the Customer cancels the purshase for security or authentication reasons.
+
+## Definitions
+
+- `IPR`: [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md)
+- `Merchant`: The one requesting a payment
```

Comment:
Sounds like we'd need to change the script that builds the website. Not sure it will be that easy and seems like an odd thing to do to save 1 click when browsing the specs on Github.

Our of interest, what was the thinking behind having the specs in a named file inside an identically named folder instead of just /0000-spec/README.md?

My suggestion would be to reconsider this now that we are publishing the specs. We should be directing the majority of people to read them on the website. Contributors and editors should be the only people browsing on Github and the experience is not that bad (it's an extra click)

Let's discuss here: https://github.com/interledger/rfcs/issues/191